README.md

```diff
@@ -4,9 +4,9 @@ testing GH merge PR bugs
 
 Alpha
```

Comment:
This line is changed in master, but diff do not indicate that (this is how we will reintroduce fixed bugs)